strings/ctype-mb.c

```diff
@@ -403,7 +403,8 @@ my_copy_fix_mb(CHARSET_INFO *cs,
   size_t fixed_length;
   size_t min_length= MY_MIN(src_length, dst_length);
 
-  well_formed_nchars= my_ci_well_formed_char_length(cs, src, src + min_length,
```

Comment:
`/server/strings/ctype-mb.c:406:66: runtime error: applying zero offset to null pointer`